iree/compiler/Conversion/LinalgToSPIRV/SplitDispatchFunctionPass.cpp

```diff
@@ -65,7 +66,11 @@ bool canSeparateOps(ArrayRef<Operation *> ops) {
   // the common case and it simplifies further analysis.
   for (auto currOp = ops.begin(), nextOp = std::next(ops.begin());
        nextOp != ops.end(); ++currOp, ++nextOp) {
-    if ((*currOp)->getNextNode() != *nextOp) return false;
+    Operation *iter = (*currOp)->getNextNode();
+    while (isa<linalg::ReshapeOp>(iter) || isa<SubViewOp>(iter)) {
```

Comment:
I think you can do something like `isa<linalg::ReshapeOp,SubViewOp>(iter)` now?

Reply:
Oh yes, thanks!